django/template/loader_tags.py

```diff
@@ -204,9 +207,11 @@ def render(self, context):
                 return template.render(context.new(values))
             with context.push(**values):
                 return template.render(context)
-        except Exception:
+        except Exception as e:
             if context.template.engine.debug:
                 raise
+            else:
+                logger.warn('template - Uncaught Exception raised during rendering.', exc_info=e)
```

Comment:
You want to either pass `exc_info=True` or `sys.exc_info()` here. `exc_info=e` only worked because it evaluated to `True`.